### DISCUSSION TITLE
Does Spotify Wrapped work with Spotube?

### DISCUSSION THREAD

**Discussion by 9182736450typist** [KRTirtho/spotube#1551] [General]
Is there a way for Spotube to send song streaming data (opt in) to spotify so Wrapped works correctly, but not connect to any other servers? As someone who anticipates **heavily** for a spotify wrapped to occur, is it possible for it to work correctly with Spotube?

**Comment by AbbadIsProcratinating** [comment]
dude spotify wrapped launched this year, check it out if its accurate and tell me too, pls
